packages/foreman/rubygem-unf/rubygem-unf.spec

```
@@ -1,83 +1,74 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name unf
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.3
-Release: 9%{?dist}
-Summary: Unicode normalization form support for Ruby/JRuby
-Group: Development/Languages
-License: MIT
+Name: rubygem-%{gem_name}
+Version: 0.1.4
+Release: 1%{?dist}
+Summary: A wrapper library to bring Unicode Normalization Form support to Ruby/JRuby
+License: 2-clause BSDL
```

Interesting license change. Need to check this, also if it's really a valid Fedora license code.

---

Looks like it changed to the standard `BSD-2-Clause` in https://github.com/knu/ruby-unf/commit/cb2061b2f081c03f5f88fc121b3c64d7e655789b (unreleased). In https://github.com/knu/ruby-unf/commit/aa75e87a14b85f651204fdc61c8ec6b2c43d6246 the official license was added. I can't find any trace of why this should be MIT. This should follow the Fedora code:
```suggestion
License: BSD
```